arviz/tests/base_tests/test_stats_ecdf_utils.py

```diff
@@ -0,0 +1,153 @@
+import pytest
+
+import numpy as np
+import scipy.stats
+from arviz.stats.ecdf_utils import (
```

Comment:
```suggestion
from ...stats.ecdf_utils import (
```

I think this is what makes CI fail